frontend/controller/actions/identity-kv.js

```diff
@@ -319,8 +289,7 @@ export default (sbp('sbp/selectors/register', {
     }
     return sbp('okTurtles.eventQueue/queueEvent', KV_QUEUE, async () => {
       const notifications = sbp('chelonia/rootState').notifications.items
-      const getUpdatedNotificationStatus = async () => {
-        const currentData = await sbp('gi.actions/identity/kv/fetchNotificationStatus')
+      const getUpdatedNotificationStatus = ({ currentData = {}, etag } = {}) => {
```

Comment:
Down below, unmodified is `getUpdatedPreferences()?.[0]`, which should at least be modified to `getUpdatedPreferences()[0]`, and any other updates as needed.